cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -101,6 +103,13 @@ def filtered_list(cls):
         if not XBlockStudioConfigurationFlag.is_enabled():
             filtered_list.append('allow_unsupported_xblocks')
 
+        # TODO: Alex Dusenbery 2017-06-22
```

Comment:
Generally, we don't put our own names or initials in the code itself.  Git history can be used to track down developers when needed.  And ownership may be transferred in the unknowing future.
Rather, for TODO statements, create a JIRA ticket and put the ticket number in the code.